packages/archsat/archsat.1.1/opam

```diff
@@ -0,0 +1,77 @@
+opam-version: "2.0"
+maintainer: "Guillaume Bury <guillaume.bury@gmail.com>"
+license: "BSD-2-clauses"
+build: [
+  [make "bin"]
+  [make "test"] { with-test }
+  [make "static"] { dedukti:installed }
+]
+install:[
+  [make
+    "MANDIR=%{man}%"
+    "BINDIR=%{archsat:bin}%"
+    "SHAREDIR=%{archsat:share}%"
+    "install"]
+  [make
+    "-C" "static"
+    "MANDIR=%{man}%"
+    "BINDIR=%{archsat:bin}%"
+    "SHAREDIR=%{archsat:share}%"
+    "install_dk"] { dedukti:installed }
+]
+remove: [
```

Comment:
The `remove` section is unnecessary since opam 2.0 (except if you need to modify files during uninstall)